.github/workflows/alert-on-pr.yml

```diff
@@ -0,0 +1,52 @@
+name: Alert on PR Changes
+
+on:
+  pull_request:
+    types: [opened, synchronize, reopened]
+
+permissions:
+  contents: read
+
+jobs:
+  alert:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+
+      - name: Set up Python
+        uses: actions/setup-python@v5
+        with:
+          python-version: '3.11'
+
+      - name: Install requests
+        run: |
+          python -m pip install --upgrade pip
+          pip install requests
+
+      - name: Send Slack alert on PR changes
+        env:
+          SLACK_WEBHOOK: ${{ secrets.DEVEX_ALERTS_SECRET }}
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
Is this token the one we use for existing alerts? Or is it a new one generated by IT?